pkg/test/framework/components/echo/kube/instance.go

```diff
@@ -329,15 +330,18 @@ func (c *instance) aggregateResponses(opts echo.CallOptions) (echo.CallResult, e
 	}
 	aggErr := istiomultierror.New()
 	for _, w := range workloads {
-		clusterName := w.(*workload).cluster.Name()
-		serviceName := fmt.Sprintf("%s (cluster=%s)", c.cfg.Service, clusterName)
-
-		out, err := common.ForwardEcho(serviceName, c, opts, w.(*workload).Client)
-		if err != nil {
-			aggErr = multierror.Append(aggErr, err)
-			continue
+		for _, ipf := range ipFamilies(w, opts) {
+			clusterName := w.(*workload).cluster.Name()
```

Comment:
nit: we can probably do the type assertion outside of the inner for loop and assign it to a variable; clusterName and serviceName can be outside of the inner loop as well.